Reply/app/src/main/java/com/example/reply/ui/navigation/ReplyNavigationActions.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.reply.ui.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Article
+import androidx.compose.material.icons.filled.Inbox
+import androidx.compose.material.icons.filled.People
+import androidx.compose.material.icons.outlined.ChatBubbleOutline
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import com.example.reply.R
+
+object ReplyRoute {
+    const val INBOX = "Inbox"
+    const val ARTICLES = "Articles"
+    const val DM = "DirectMessages"
+    const val GROUPS = "Groups"
+}
+
+data class ReplyTopLevelDestination(
+    val route: String,
+    val selectedIcon: ImageVector,
+    val unselectedIcon: ImageVector,
+    val iconTextId: Int
+)
+
+class ReplyNavigationActions(private val navController: NavHostController) {
```

Comment:
I think it can be extended to add more actions in updates like -> navigateToCreateEmail etc or extension approach is better for future actions as well ?